modules/configuring-your-ide.adoc

```diff
@@ -0,0 +1,47 @@
+:_module-type: PROCEDURE
+
+[id="configuring-your-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring your code-server workbench
+Incorporating standard database clients into your code-server workbench can streamline your workflow, offering a seamless interface for managing databases. You can incorporate different Database Management System (DBMS) clients into your code-server workbench by using built-in extensions. 
+
+For more information on code-server, see link:https://github.com/coder/code-server[code-server in GitHub].
+
+=== Incorporating DBMS clients using extensions
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have created a data science project that has a code-server workbench.
+
+.Procedure
+
+. From the {productname-short} dashboard, click *Data Science Projects*.
++
+The *Data science projects* page opens.
+. Click the name of the project containing the code-server workbench you want to start.
++
+The *Details* page for the project opens.
+. Click the toggle in the *Status* column for the relevant workbench to start a workbench that is not running.
++
+The status of the workbench that you started changes from *Stopped* to *Running*. 
+. After the workbench has started, click *Open* to open the workbench notebook.
+. In the Activity Bar, click the Extensions icon. (image:images/codeserver-extensions-icon.png[The Extensions icon])
+. Search for the name of the extension you want to install. 
+. Click *Install* to add the extension to your code-server environment.
++ 
+The extension you installed appears in the *Browser - Installed* list on the *Extensions* panel.
+
+==== Extensions
```

Comment:
The PR from @atheo89 had some suggestions which extensions to install for which DB. This seems useful, not sure why it was removed

* https://github.com/opendatahub-io/notebooks/pull/378/files

Reply:
https://github.com/opendatahub-io/opendatahub-documentation/pull/150#discussion_r1463279388

Reply:
good point; still, the docs is really weird at this point; it should be presented as a guide to install vscode extensions in general, there is nothing specific to database clients in it by now

Reply:
How about 'You can use built-in extensions to streamline your workflow, add new languages, themes, debuggers, and connect to additional services.

For more information on code-server, see link:https://github.com/coder/code-server[code-server in GitHub].'

Reply:
> built-in extensions

built-in? isn't the point of the extension marketplace precisely that the features need not be built-in, but can be installed on-demand?

other than that, it sounds better to me.

I would like to understand where the initial focus on databases comes from. I will ask on internal slack for the product requirement that leads to this.